src/Game/Select.c

```diff
@@ -1303,7 +1303,7 @@ void selStatusDraw0(Ship *ship)
         fontMakeCurrent(selGroupFont0);
         fontPrint(x - halfWidth - fontWidth("C") - 1, rect.y1 + 5 - fontHeight(" "), selHotKeyNumberColor, "C");
     }
-    if((ship->shiptype == MissileDestroyer) || (ship->shiptype == MinelayerCorvette))      //ship has missles, so display missile status
+    if((ship->shiptype == MissileDestroyer) || (ship->shiptype == MinelayerCorvette) || (ship->shiptype == P1MissileCorvette))      //ship has missles, so display missile status
```

Comment:
[M] Not ideal considering the scalability concern (#2).

Some alternatives:
1. A check function on the fly, which might imply perf issues.
2. A top-level indicator attribute ([`Ammo`](https://www.modenc.renegadeprojects.com/Ammo) from _Command and Conquer_ titles) which might not scale well.

Reply:
Resolved by 796629a